caffe2/contrib/aten/aten_op_template.h

```diff
@@ -179,8 +179,9 @@ class ATenOp : public Operator<Context> {
     std::vector<std::string> attrs;
     for (const auto i : c10::irange(operator_def.arg_size())) {
       auto & attr = operator_def.arg(i);
-      if(attr.name() == "operator" || attr.name() == "type" )
+      if(attr.name() == "operator" || attr.name() == "type" || attr.name() == "overload_name" ) {
```

Comment:
Hmm, did you get an ack from someone who maintains caffe2 codebase? (Though not sure who owns it right now)
@ezyang do you know by any change?

Reply:
Not really, but this is back compatible. `overload_name` is an optional attribute which is ignored by Caffe2 with this exact condition :)